Podfile.lock

```diff
@@ -34,12 +34,12 @@ SPEC REPOS:
 SPEC CHECKSUMS:
   AccessibilitySnapshot: a91e4a69f870188b51f43863d9fc7269d07cdd93
   GliaCoreDependency: 37f48a5a32e2646617b87cbe9d4b30eedd123f6f
-  GliaCoreSDK: 088d9a33b1b96264bd713d7375005ca4939b390d
+  GliaCoreSDK: 047de3f34a89010b66960b984dea7385bd76bc3a
   SnapshotTesting: 6141c48b6aa76ead61431ca665c14ab9a066c53b
   SwiftLint: c1de071d9d08c8aba837545f6254315bc900e211
   TwilioVoice: 9563c9ad71b9ab7bbad0b59b67cfe4be96c75d23
   WebRTC-lib: 4e9a17058f880cd658e88383c1ac8f1119af3700
 
 PODFILE CHECKSUM: 40bea65dec1e511d94b8bce63fe030eae8e04f7b
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.15.0
```

Comment:
Would it be better to upgrade CocoaPods locally?

Reply:
updated

Reply:
4c729f0fdd1c65977b4d451b79eeed9c87c8dd7b